### ISSUE TITLE
Compilation does not work as in the documentation

### ISSUE THREAD

**Issue by oleg-alexandrov** [CGAL/cgal#4413]
# Issue Details

I followed the instructions at:

https://github.com/CGAL/cgal/blob/master/INSTALL.md

So I did:

 cmake .. -DCMAKE_BUILD_TYPE=Release -DCGAL_DIR=/home/oalexan1/projects/CGAL

It says:

Manually-specified variables were not used by the project:

    CGAL_DIR

and compilation failed.

Only in the second section, the one named "Building a Program Using CGAL", at the bottom it is stated that one should do:

/path/to/cgal.git/Scripts/scripts/cgal_create_cmake_script

and even here there is a typo, it should be:

/path/to/cgal.git/scripts/cgal_create_cmake_script

I will suggest that the "NOTICE" section should be wiped since it suggests that no linking is necessary which is wrong, as MPRF is needed, and since it does not mention running cgal_create_script, and since it suggests using -DCGAL_DIR, which on its own is not helpful.

Also, the documentation suggests the user will want to build their code inside the source code tree of CGAL, which is the opposite of what is suggested, the builds must always be outside of the source code, at least that is the CMake philosophy.

**Comment by afabri** [comment]
The path to `cgal_create_cmake_script` is correct as on the page INSTALL.md we describe the build process in case you have  git clone`ed the CGAL repository.  Note that it has another layout of files than a tarball.

**Comment by oleg-alexandrov** [comment]
Thank you. I guess a simple solution is to add the text:

```
/path/to/cgal.git/Scripts/scripts/cgal_create_cmake_script

(or /path/to/cgal.git/scripts/cgal_create_cmake_script if using the zipped source code archive.)

```
These are all minor glitches, and by reading the documentation a few times and fooling around some more eventually one figures it out, it is just if the doc is a bit more clear it is easier on the user, especially if the user is new to the software.

**Comment by afabri** [comment]
We appreciate your feedback.

**Comment by oleg-alexandrov** [comment]
I put a pull request in https://github.com/CGAL/cgal/pull/4419

**Comment by lrineau** [comment]
Fixed by #4419.
